input/system/selfhosted/log_receiver.go

```diff
@@ -76,7 +76,7 @@ func DiscoverLogLocation(servers []*state.Server, globalCollectionOpts state.Col
 		}
 
 		if logDestination == "syslog" {
-			prefixedLogger.PrintInfo("Log location detected as syslog - please check our setup guide for rsyslogd or syslog-ng instructions")
+			prefixedLogger.PrintInfo("WARNING: Logging via syslog - please check our setup guide for rsyslogd or syslog-ng instructions")
```

Comment:
I think WARNINGs here make it clear you need to change something, but don't imply the setup is broken.